### PR TITLE
Add integration tests for adding scopes into URL in implicit grant

### PR DESCRIPTION
**Pull request by chashikajw**
## Purpose
As $subject, this will add the integrations test for adding scopes into URL in the implicit grant.

### Related PRs
1. https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1655

### Issues
1. https://github.com/wso2/product-is/issues/12319

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java [hunk 1]
```diff
@@ -214,7 +217,11 @@ public void testSendApprovalPost() throws Exception {
 
 		accessToken = DataExtractUtil.extractParamFromURIFragment(locationHeader.getValue(),
 				OAuth2Constant.ACCESS_TOKEN);
+		String urlScopes = DataExtractUtil.extractParamFromURIFragment(locationHeader.getValue(),
+				OAuth2Constant.SCOPE_PLAYGROUND_NAME);
 		Assert.assertNotNull(accessToken, "Access token is null.");
+		Assert.assertNotNull(accessToken, "Scope is null.");
```
Comment (janakamarasena) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java:223]: Unwanted assertion

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java [hunk 2]
```diff
@@ -106,6 +107,7 @@ public void testInit() throws Exception {
 
 		setSystemproperties();
 		client = HttpClientBuilder.create().build();
+		scopes = "scopeTest";
```
Comment (janakamarasena) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java:110]: lets make it like `abc`
```suggestion
		scopes = "abc";
```

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java [hunk 3]
```diff
@@ -214,7 +217,11 @@ public void testSendApprovalPost() throws Exception {
 
 		accessToken = DataExtractUtil.extractParamFromURIFragment(locationHeader.getValue(),
 				OAuth2Constant.ACCESS_TOKEN);
+		String urlScopes = DataExtractUtil.extractParamFromURIFragment(locationHeader.getValue(),
+				OAuth2Constant.SCOPE_PLAYGROUND_NAME);
```
Comment (janakamarasena) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java:221]: Use the `OAuth2Constant.OAUTH2_SCOPE` constant instead

```suggestion
				OAuth2Constant.OAUTH2_SCOPE);
```

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java (hunk 2 above)
Comment (chashikajw) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java]: Fixed

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java (hunk 3 above)
Comment (chashikajw) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java]: Fixed

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java (hunk 1 above)
Comment (chashikajw) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java]: Fixed

**Review by janakamarasena — APPROVED** [review]

**Pull request merged**